src/main/java/prefs/UserConfig.java

```diff
@@ -0,0 +1,19 @@
+package prefs;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Test class to see if the file handler works with other config subclasses in the new framework
+ */
+public class UserConfig implements Config {
```

Comment:
While this is the feature to be made in the near future, I think mixing it with this PR may create confusion. Maybe stash this for now.

If you would like to test if the `ConfigFile` works with other config, maybe create a private class in the `ConfigFile` test class? Then `UserConfigTest` might be generalised to `ConfigFileTest` for now.